Examples/StereoKitTest/StereoKitTest_UWP/StereoKitTest_UWP.csproj

```diff
@@ -106,11 +102,24 @@
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
+  <!-- Make sure the UWP binaries are built and included. -->
+  <PropertyGroup>
+    <SKCMakePreset Condition="'$(Platform)' == 'ARM'"  >Uwp_Arm32_$(Configuration)</SKCMakePreset>
+    <SKCMakePreset Condition="'$(Platform)' == 'ARM64'">Uwp_Arm64_$(Configuration)</SKCMakePreset>
+    <SKCMakePreset Condition="'$(Platform)' == 'x64'"  >Uwp_x64_$(Configuration)</SKCMakePreset>
+    <PreBuildEvent>cd $(ProjectDir)..\..\..\ &amp; cmake --preset $(SKCMakePreset) &amp; cmake --build --preset $(SKCMakePreset)</PreBuildEvent>
+  </PropertyGroup>
+  <Target Name="SKTestRebuildUWP" AfterTargets="Clean">
+    <RemoveDir Directories="$(ProjectDir)..\..\..\bin\intermediate\UWP_$(Platform)_$(Configuration)" />
```

Comment:
MSVC does build directories a little differently than other compilers! Basically, instead of generating entirely new projects for Release vs. Debug like clang/gcc, msvc will just add a Debug or Release folder for binaries and intermediates inside the same project.